app/src/main/res/values-b+zh+CN/localized.xml

```diff
@@ -65,6 +65,9 @@
     <string name="skill_maker_quick">迅击</string>
     <string name="skill_maker_arts">技击</string>
     <string name="skill_maker_buster">力击</string>
+    <string name="skill_maker_kukulcan">Kukulcan</string>
```

Comment:
Don't use English values in other languages, ignore the warnings about missing translations instead.

The app will display the English values if the translated ones are missing.